app/src/main/res/drawable/rounded_check_box.xml

```diff
@@ -0,0 +1,28 @@
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
why attributes have **hardcode** please change to resource value

Reply:
Done